po/README.md

```diff
@@ -340,4 +340,49 @@ Usage for git-po-helper:
 
 Run "git-po-helper" without arguments to show usage.
 
+
+Conventions
+-----------
+
+There are some conventions that l10n contributors must follow:
+
+1. The subject of each l10n commit should be prefixed with "l10n: ".
+2. Do not use non-ASCII characters in the subject of a commit.
+3. The length of commit subject (first line of the commit log) should
+   be less than 50 characters, and the length of other lines of the
+   commit log should be no more than 72 characters.
+4. Add "Signed-off-by" trailer to your commit log, like other commits
+   in Git. You can automatically add the trailer by committing with
+   the following command:
+
+        git commit -s
```

Comment:
Add a blank line here, just like the next item?

Reply:
fixed.